02_configure_host.sh

```diff
@@ -53,53 +53,48 @@ if [[ $OS == ubuntu ]]; then
   source disable_apparmor_driver_libvirtd.sh
 else
   if [ "$MANAGE_PRO_BRIDGE" == "y" ]; then
-      case $VERSION_ID in
-        8)
-          NMC="NM_CONTROLLED=no\n"
-          ;;
-        9)
-          NMC=""
-          ;;
-      esac
       # Adding an IP address in the libvirt definition for this network results in
       # dnsmasq being run, we don't want that as we have our own dnsmasq, so set
       # the IP address here
-      if [ ! -e /etc/sysconfig/network-scripts/ifcfg-provisioning ] ; then
+      if [ ! -e /etc/NetworkManager/system-connections/provisioning.nmconnection ] ; then
         if [[ "${PROVISIONING_IPV6}" == "true" ]]; then
-          echo -e "DEVICE=provisioning\nTYPE=Bridge\nONBOOT=yes\n${NMC}IPV6_AUTOCONF=no\nIPV6INIT=yes\nIPV6ADDR=$PROVISIONING_IP/$PROVISIONING_CIDR" | sudo dd of=/etc/sysconfig/network-scripts/ifcfg-provisioning
```

Comment:
> This looks really nice! Good job!
> 
> Do we have any tests that check the IPv6 config? I think the normal pipelines use IPv4 so it would be nice to verify.

I did not checked with ipv6 yes. Thanks for pointing out. I will start checking it with ipv6